input/metadata/properties/tasks_properties.json

```
@@ -50,6 +50,12 @@
           "description": "The path to the second metadata properties JSON file to load and compare to the other metadata file loaded",
           "default": "input/metadata/properties/default.json"
         },
+        "convert_variable_table_path": {
+          "type": "string",
+          "pattern": "^.*.csv$",
+          "description": "The path to the CSV file that contains the previous and the new name of variables.",
+          "default": ""
```

```suggestion
          "default": null,
          "nullable": true,
```
Should this also be nullable? In this case I think it would make sense that E2E testing could run fine with or without this.